PocketKit/Sources/Sync/Operations/FetchArchive.swift

```diff
@@ -64,19 +61,19 @@ class FetchArchive: SyncOperation {
     }
 
     private func fetchArchive() async throws {
-        var pagination = PaginationSpec(maxItems: maxItems)
+        var pagination = PaginationSpec(maxItems: SyncConstants.Archive.firstLoadMaxCount, pageSize: SyncConstants.Archive.initalPageSize)
 
         repeat {
             let result = try await fetchPage(pagination)
 
             if case .started = initialDownloadState.value,
                let totalCount = result.data?.user?.savedItems?.totalCount,
                pagination.cursor == nil {
-                initialDownloadState.send(.paginating(totalCount: totalCount))
+                initialDownloadState.send(.paginating(totalCount: totalCount > pagination.maxItems ? pagination.maxItems : totalCount))
```

Comment:
style: what about using `min(totalCount, pagination.maxItems)` so we can understand better that we want the smaller count?